nerfstudio/process_data/images_to_nerfstudio_dataset.py

```diff
@@ -85,11 +88,11 @@ def main(self) -> None:
                     same_dimensions=self.same_dimensions,
                     keep_image_dir=True,
                 )
-                image_rename_map_paths.update(eval_image_rename_map_paths)
+                image_rename_map_eval = dict(
```

Comment:
just to follow the name convention, could we rename it to `eval_image_rename_map`?